ServerCore/Pages/Events/Delete.cshtml

```diff
@@ -18,127 +18,127 @@
     <hr />
     <dl class="dl-horizontal">
         <dt>
-            @Html.DisplayNameFor(model => model.Event.Name)
+            Name
         </dt>
         <dd>
-            @Html.DisplayFor(model => model.Event.Name)
+            @Model.Event.Name
         </dd>
         <dt>
-            @Html.DisplayNameFor(model => model.Event.UrlString)
+            UrlString
```

Comment:
You didn't break anything here, but it looks like something is off WRT UrlString and HomePartial as they do not match the dt-followed-by-dd pattern used by the other properties. Not an important fix as we rarely delete events anyway...